userprofile/templates/userprofile/profile.html

```diff
@@ -12,62 +12,60 @@
 {% endblock %}
 
 {% block content %}
-<div class="page-header">
-    <h1>
-        {% if user.first_name or user.last_name %}
-        {{ user.first_name }} {{ user.last_name }}
-        {% else %}
-        {% trans "[Unknown name]" %}
-        {% endif %}
-    </h1>
+<div class="row">
+	<div class="col-md-offset-2 col-md-2 page-header">
+		<div class="gravatar">
+			<a href="http://gravatar.com" target="_blank" rel="noopener noreferrer">
+				<img class="img-circle" src="{{ profile.avatar_url }}" alt="" aria-label="Change your avatar">
+				<span class="notify-badge">Change</span>
+			</a>
+		</div>
+		<h1>
+			{% if user.first_name or user.last_name %}
+			{{ user.first_name }} {{ user.last_name }}
+			{% else %}
+			{% trans "[Unknown name]" %}
+			{% endif %}
+		</h1>
+		<h3 style="margin-top: -10px;" class="text-muted">{% firstof user.username miss %}</h3>
```

Comment:
The variable `miss` is not yet defined at this point so I think it will always render the empty string here. The variable could be defined earlier in the code or you can use a literal string parameter in the `firstof` tag.